src/main/java/org/killbill/billing/payment/api/PaymentApi.java

```diff
@@ -365,7 +365,7 @@ public Payment createCreditWithPaymentControl(Account account, UUID paymentMetho
      * @return the payment
      * @throws PaymentApiException
      */
-    public Payment getPayment(UUID paymentId, boolean withPluginInfo, Iterable<PluginProperty> properties, TenantContext context)
+    public Payment getPayment(UUID paymentId, boolean withPluginInfo, boolean withAttempts, Iterable<PluginProperty> properties, TenantContext context)
```

Comment:
We should probably make all APIs symmetric and update other methods accordingly (`getAccountPayments`, etc.).